packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -250,6 +251,53 @@ describe('babel-plugin-strip-runtime using source code', () => {
 
     describe('with jsx pragma', () => {
       it('work with classic jsx pragma', () => {
+        const codeWithPragma = `
+          /** @jsx jsx */
+          import { css, jsx } from '@compiled/react';
+
+          const Component = () => (
+            <div css={{ fontSize: 12, color: 'blue' }}>
+              hello world 2
+            </div>
+          );
+
+          const Component2 = () => (
+            <div css={css({ fontSize: 12, color: 'pink' })}>
+              hello world 2
+            </div>
+          );
+        `;
+
+        const actual = transform(codeWithPragma, {
+          run: 'both',
+          runtime: 'classic',
+        });
+
+        expect(actual).toMatchInlineSnapshot(`
+          "/* app.tsx generated by @compiled/babel-plugin v0.0.0 */
```

Comment:
And then for the snapshot. Is the assertion that the jsx pragma is removed and replaced by React? Can we assert that?